Mathlib/Order/ConditionallyCompleteLattice/Basic.lean

```diff
@@ -219,16 +219,12 @@ complete linear orders, we prefix `sInf` and `sSup` by a `c` everywhere. The sam
 hold in both worlds, sometimes with additional assumptions of nonemptiness or
 boundedness. -/
 class ConditionallyCompleteLinearOrderBot (α : Type*) extends ConditionallyCompleteLinearOrder α,
-  Bot α where
-  /-- `⊥` is the least element -/
-  bot_le : ∀ x : α, ⊥ ≤ x
-  /-- The supremum of the empty set is `⊥` -/
+  OrderBot α where
```

Comment:
To distinguish this from the fields below, just like we distinguish hypotheses from proofs:
```suggestion
    OrderBot α where
```

bors d+